lib/presentation/pages/checkout/checkout_page.dart

```diff
@@ -0,0 +1,127 @@
+import 'package:flutter/material.dart';
+import 'package:furniture_shop/generated/assets/fonts.gen.dart';
+import 'package:furniture_shop/presentation/pages/checkout/widgets/card_shadow_widget.dart';
+import 'package:furniture_shop/presentation/pages/checkout/widgets/info_price_widget.dart';
+import 'package:furniture_shop/presentation/pages/checkout/widgets/payment_widget.dart';
+import 'package:furniture_shop/presentation/pages/checkout/widgets/shipping_address_widget.dart';
+import 'package:furniture_shop/presentation/widgets/base/custom_appbar.dart';
+import 'package:furniture_shop/presentation/widgets/base/custom_button.dart';
+import 'package:furniture_shop/presentation/widgets/base/custom_text.dart';
+import 'package:furniture_shop/values/colors.dart';
+import 'package:furniture_shop/values/dimens.dart';
+import 'package:furniture_shop/values/font_sizes.dart';
+
+class CheckOutPage extends StatelessWidget {
+  const CheckOutPage({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: _buildAppBar(),
+      body: _buildBody(),
```

Comment:
Add to scroll screen becuz if using small screen, the content cannot be seen

Reply:
done